### PR TITLE
Feature/tcloud 2861 2

### PR DESCRIPTION
**Pull request by ephox-mogran**
Recreating https://github.com/tinymce/tinymce-docs/pull/1041/files on behalf of mhideo on the new branch.

### CODE REVIEW

File: _includes/release-notes/overview.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ TinyMCE 5.0 has an ARIA [WCAG 2.1 standards](https://www.w3.org/WAI/standards-gu
 Include this line of code in the <head> of the HTML page and link to the tinymce.min.js source file:
 
 ```js
-<script src="{{ site.cdnurl }}?apiKey=your_API_key"></script>
+<script src="{{ site.cdnurl }}"></script>
```
Comment (ephox-mogran) [inline _includes/release-notes/overview.md]: @mhideo This one doesn't explain `no-api-key` that I can see.

File: cloud-deployment-guide/editor-and-features.md [hunk 2]
```diff
@@ -80,7 +90,7 @@ Migrating from a self-hosted environment to Tiny Cloud is easy. Remove the exist
 Replace the script tag with the following:
 
 ```js
-<script src="{{ site.cdnurl }}?apiKey=your_API_key"></script>
+<script src="{{ site.cdnurl }}" referrerpolicy="origin"></script>
```
Comment (ephox-mogran) [inline cloud-deployment-guide/editor-and-features.md]: This one doesn't mention no-api-key's meaning.

File: cloud-deployment-guide/editor-plugin-version.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&mentions=2.0.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&mentions=2.0.0" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]: `src` `?&` should just be `?`
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?mentions=2.0.0" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 4]
```diff
@@ -81,7 +81,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&tinydrive=1.1.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&tinydrive=1.1.0" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?tinydrive=1.1.0" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 5]
```diff
@@ -92,7 +92,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&tinycomments=2.0.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&tinycomments=2.0.0" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?tinycomments=2.0.0" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 6]
```diff
@@ -103,7 +103,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&pageembed=1.0.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&pageembed=1.0.0" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?pageembed=1.0.0" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 7]
```diff
@@ -114,7 +114,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&permanentpen=1.0.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&permanentpen=1.0.0" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?permanentpen=1.0.0" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 8]
```diff
@@ -125,7 +125,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&formatpainter=1.0.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&formatpainter=1.0.0" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?formatpainter=1.0.0" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 9]
```diff
@@ -136,7 +136,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&powerpaste=4.0.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&powerpaste=4.0.0" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?powerpaste=4.0.0" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 10]
```diff
@@ -147,7 +147,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&tinymcespellchecker=1.0.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&tinymcespellchecker=1.0.0" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?tinymcespellchecker=1.0.0" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 11]
```diff
@@ -158,7 +158,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&a11ychecker=2.0.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&a11ychecker=2.0.0" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?a11ychecker=2.0.0" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 12]
```diff
@@ -169,7 +169,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&advcode=2.0.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&advcode=2.0.0" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?advcode=2.0.0" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 13]
```diff
@@ -180,7 +180,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&mediaembed=2.0.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&mediaembed=2.0.0" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?mediaembed=2.0.0" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 14]
```diff
@@ -191,7 +191,7 @@ Use the URL query parameters to specify the version of each premium plugin. This
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&linkchecker=2.0.0"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&linkchecker=2.0.0" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?linkchecker=2.0.0" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 15]
```diff
@@ -209,7 +209,7 @@ The "SDK" version lets the TinyMCE Plugin Manager know that you're not using Tin
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&mentions=sdk"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&mentions=sdk" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?mentions=sdk" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 16]
```diff
@@ -219,7 +219,7 @@ The "SDK" version lets the TinyMCE Plugin Manager know that you're not using Tin
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&tinydrive=sdk"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&tinydrive=sdk" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?tinydrive=sdk" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 17]
```diff
@@ -229,7 +229,7 @@ The "SDK" version lets the TinyMCE Plugin Manager know that you're not using Tin
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&tinycomments=sdk"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&tinycomments=sdk" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?tinycomments=sdk" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 18]
```diff
@@ -239,7 +239,7 @@ The "SDK" version lets the TinyMCE Plugin Manager know that you're not using Tin
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&pageembed=sdk"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&pageembed=sdk" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?pageembed=sdk" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 19]
```diff
@@ -249,7 +249,7 @@ The "SDK" version lets the TinyMCE Plugin Manager know that you're not using Tin
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&permanentpen=sdk"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&permanentpen=sdk" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?permanentpen=sdk" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 20]
```diff
@@ -259,7 +259,7 @@ The "SDK" version lets the TinyMCE Plugin Manager know that you're not using Tin
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&formatpainter=sdk"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&formatpainter=sdk" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?formatpainter=sdk" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 21]
```diff
@@ -269,7 +269,7 @@ The "SDK" version lets the TinyMCE Plugin Manager know that you're not using Tin
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&powerpaste=sdk"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&powerpaste=sdk" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?powerpaste=sdk" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 22]
```diff
@@ -279,7 +279,7 @@ The "SDK" version lets the TinyMCE Plugin Manager know that you're not using Tin
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&tinymcespellchecker=sdk"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&tinymcespellchecker=sdk" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?tinymcespellchecker=sdk" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 23]
```diff
@@ -289,7 +289,7 @@ The "SDK" version lets the TinyMCE Plugin Manager know that you're not using Tin
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&a11ychecker=sdk"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&a11ychecker=sdk" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?a11ychecker=sdk" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 24]
```diff
@@ -299,7 +299,7 @@ The "SDK" version lets the TinyMCE Plugin Manager know that you're not using Tin
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&advcode=sdk"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&advcode=sdk" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?advcode=sdk" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 25]
```diff
@@ -309,7 +309,7 @@ The "SDK" version lets the TinyMCE Plugin Manager know that you're not using Tin
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&mediaembed=sdk"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&mediaembed=sdk" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?mediaembed=sdk" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/editor-plugin-version.md [hunk 26]
```diff
@@ -319,7 +319,7 @@ The "SDK" version lets the TinyMCE Plugin Manager know that you're not using Tin
 ##### Example
 
 ```js
-<script src="https://cloud.tinymce.com/5/plugins.min.js?apiKey=your_API_key&linkchecker=sdk"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?&linkchecker=sdk" referrerpolicy="origin"></script>
```
Comment (tjdett) [inline cloud-deployment-guide/editor-plugin-version.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?linkchecker=sdk" referrerpolicy="origin"></script>
```

File: cloud-deployment-guide/plugin-editor-version-compatibility.md [hunk 27]
```diff
@@ -11,6 +11,11 @@ Specific versions of TinyMCE are compatible with premium plugins.
 Use the following matrix in conjunction with the optional parameters for specifying editor version and [plugin version]({{ site.baseurl }}/cloud-deployment-guide/editor-plugin-version) to ensure compatibility.
 
 > Important: Ensure the plugins are supported in your editor version when deploying premium plugins only[^footnote] via [Tiny Cloud]({{ site.baseurl }}/cloud-deployment-guide/features-only).
+> Also ensure you are using the latest script tag and substitute 'no-api-key' with your api key.
+```js
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/tinymce.min.js" referrerpolicy="origin></script>
```
Comment (tjdett) [inline cloud-deployment-guide/plugin-editor-version-compatibility.md]:
```suggestion
<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/tinymce.min.js" referrerpolicy="origin"></script>
```

File: migration-from-4x.md [hunk 28]
```diff
@@ -18,13 +18,13 @@ This chapter describes the migration process and workarounds for customers using
 
 To serve TinyMCE 5 from the cloud, include this in your html page:
 ```js
-<script src="{{ site.cdnurl }}?apiKey=your_API_key"></script>
+<script src="{{ site.cdnurl }}" referrerpolicy="origin"></script>
```
Comment (ephox-mogran) [inline migration-from-4x.md]: We don't explain no-api-key here.

File: quick-start.md [hunk 29]
```diff
@@ -16,7 +16,7 @@ TinyMCE 5.0 is perfect for developers who want to see how the new version of Tin
 Include this line of code in the `<head>` of the HTML page and link to the tinymce.min.js source file:
 
 ```html
-<script src="{{ site.cdnurl }}?apiKey=your_API_key"></script>
+<script src="{{ site.cdnurl }}" referrerpolicy="origin"></script>
```
Comment (ephox-mogran) [inline quick-start.md]: We don't explain no-api-key here.

File: cloud-deployment-guide/editor-plugin-version.md [hunk 30]
```diff
@@ -13,7 +13,7 @@ Use the URL provided to specify the TinyMCE version when deploying via Tiny Clou
 The following example is the default for loading TinyMCE 5.0 via Tiny Cloud:
 
 ```js
```
Comment (ephox-mogran) [inline cloud-deployment-guide/editor-plugin-version.md]: We probably need to explain no-api-key at least once on this page.

File: quick-start.md [hunk 31]
```diff
@@ -34,7 +34,7 @@ In this example, replace `<textarea id='mytextarea'>` with a TinyMCE 5.0 editor
 <!DOCTYPE html>
 <html>
 <head>
-  <script src='{{ site.cdnurl }}?apiKey=your_API_key'></script>
+  <script src='{{ site.cdnurl }}' referrerpolicy="origin></script>
```
Comment (ephox-mogran) [inline quick-start.md]: Missing "

File: cloud-deployment-guide/editor-and-features.md (hunk 2 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-and-features.md]: fixed:  http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-and-features/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 3 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed:  http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-and-features/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 4 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed:  http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-and-features/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 5 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed:  http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-and-features/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 7 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 8 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 9 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 10 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 11 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 12 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 13 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 14 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 15 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 16 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 17 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 18 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 19 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 20 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 21 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 22 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 23 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 24 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 26 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: fixed: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: cloud-deployment-guide/plugin-editor-version-compatibility.md (hunk 27 above)
Comment (mhideo) [reply cloud-deployment-guide/plugin-editor-version-compatibility.md]: I put this together:  

http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/plugin-editor-version-compatibility/

File: quick-start.md (hunk 29 above)
Comment (mhideo) [reply quick-start.md]: added: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/quick-start/

File: migration-from-4x.md (hunk 28 above)
Comment (mhideo) [reply migration-from-4x.md]: added:  http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/migration-from-4x/

File: cloud-deployment-guide/editor-plugin-version.md (hunk 30 above)
Comment (mhideo) [reply cloud-deployment-guide/editor-plugin-version.md]: done:  http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/cloud-deployment-guide/editor-plugin-version/

File: quick-start.md (hunk 31 above)
Comment (mhideo) [reply quick-start.md]: fixed http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/quick-start/

File: _includes/release-notes/overview.md (hunk 1 above)
Comment (mhideo) [reply _includes/release-notes/overview.md]: Added: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/release-notes/release-notes50/#overview

**Pull request merged**